views/stylesheets/application.scss

```diff
@@ -2,12 +2,13 @@
 
 $black: black;
 $white: white;
-$primary: #4793e3;
+$primary: #4792e39c;
```

Comment:
Did you mean to change this? This isn't a valid hex value.

Reply:
Yes, it changes to that when I bump down the opacity.